### DISCUSSION TITLE
Append custom to release notes

### DISCUSSION THREAD

**Discussion by reddec** [goreleaser/goreleaser#1560] [Q&A] [answered]
Good day!

Is any way to append information to the release notes without total replacement (`--release-notes` flag)?

**Core reason:**

I would like to add information about repository to the release notes. For example, in [my project](https://github.com/reddec/trusted-cgi) I would like to add note about installation in a same way as instructions for docker added:

```
sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
echo "deb https://dl.bintray.com/reddec/debian all main" | sudo tee -a /etc/apt/sources.list
sudo apt update
sudo apt install trusted-cgi
```

**Comment by caarlos0** [comment, marked as answer]
Hey

check `goreleaser release --help`, you can add both a custom header and footer :)

**Comment by reddec** [comment]
Thanks!

For future googlers:

* footer: ```goreleaser release --release-footer footer.md```
* header: ```goreleaser release --release-header header.md```
